### DISCUSSION TITLE
ItoMCacheNear meaning?

### DISCUSSION THREAD

**Discussion by UESTCNickYoung** [opcm/pcm#499] [Q&A]
Hello, I run the pcm-pcie on my server, which has  an IceLake CPU: Intel(R) Xeon(R) Silver 4314 CPU @ 2.40GHz.
The pcm-pcie output looks like this:
Skt |  PCIRdCur  |  ItoM  |  ItoMCacheNear  |  UCRdF  |  WiL  |  PCIe Rd (B)  |  PCIe Wr (B)  
 1       20 M       19 M        1219 K           0       91 K      1293 M          1313 M     (Total)
 1      148 K       16 M        1219 K           0       91 K      9528 K          1149 M     (Miss)
 1       20 M     2574 K         244             0        0        1283 M           164 M     (Hit)

I know the meaning of ItoM, but I can't find the definition of ItoMCacheNear in any document of pcm-pcie.  
I want to know the meaning of ItoMCacheNear event. Besides, there is no ItoMCacheNear event on my Skylake Server, does this mean the ItoMCacheNear event is specific  for IceLake CPU?

Hope for answering!

**Comment by rdementi** [comment]
the event database is a good source of information:

https://github.com/intel/perfmon/blob/e5799ad8be1b7a251901a9f39b582536bae87127/ICX/events/icelakex_uncore.json#L4083

this is an I/O partial write request event
